src/screens/home.tsx

```diff
@@ -1,17 +1,125 @@
-import { StyleSheet, View } from 'react-native'
+import { useState } from 'react'
+import { StyleSheet, View, Linking, FlatList } from 'react-native'
 import { Button, Text } from 'react-native-paper'
+import * as repl from 'repl'
+import { config } from '../config'
 import { ScreenNavigationProps } from '../routes'
+import SelectDropdown from 'react-native-select-dropdown'
 
 type HomeScreenProps = ScreenNavigationProps<'Home'>
 
-const HomeScreen = ({ navigation }: HomeScreenProps) => (
-  <View style={styles.container}>
-    <Text style={styles.text}>Home Screen</Text>
-    <Button mode="contained" onPress={() => navigation.navigate('Details')}>
-      Go to details
-    </Button>
-  </View>
-)
+const tramStops = [
+  "St Peter''s Square",
+  'Chorlton',
+  'Old Trafford',
+  'Cornbrook',
+  'Firswood',
+]
+
+interface TFGMResponse {
+  [index: string]: string
+  Dest0: string
+  Dest1: string
+  Dest2: string
+  Dest3: string
+  Wait0: string
+  Wait1: string
+  Wait2: string
+  Wait3: string
+}
```

Comment:
Do you actually need the `DestN` and `WaitN` properties here? What happens if you remove them?

If you're defining an object type that has string keys and string values, you can just say `Record<string, string>`. So here you can do

```suggestion
type TFGMResponse = Record<string, string>
```